cmd/jaas/cmd/destroy_controller.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2025 Canonical.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/juju/cmd/v3"
+	"github.com/juju/gnuflag"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/jujuclient"
+
+	jimmAPI "github.com/canonical/jimm/v3/pkg/api"
+	"github.com/canonical/jimm/v3/pkg/api/params"
+)
+
+const (
+	destroyControllerDoc = `
+
+`
+	destroyControllerExamples = `
+	juju [jaas] destroy-controller <controller name>
+	juju [jaas] bootstrap mycontroller
+`
+)
+
+// destroyControllerCommand starts a bootstrap jobon the controller.
+type destroyControllerCommand struct {
+	modelcmd.ControllerCommandBase
+	out cmd.Output
+
+	store jujuclient.ClientStore
+
+	controllerName string
+
+	// Flags
+	detach bool
+}
+
+// NewDestroyControllerStartCommand returns a command to start a job
+// that will destroy a Juju controller.
+func NewDestroyControllerStartCommand() cmd.Command {
+	cmd := &destroyControllerCommand{
+		store: jujuclient.NewFileClientStore(),
+	}
+
+	return modelcmd.WrapBase(cmd)
+}
+
+// Init implements modelcmd.Command.
+func (c *destroyControllerCommand) Init(args []string) error {
+	if len(args) < 1 {
+		return fmt.Errorf("missing controller name")
+	}
+
+	c.controllerName, args = args[0], args[1:]
+	if len(args) > 0 {
+		return fmt.Errorf("unknown arguments")
+	}
+	return nil
+}
+
+// SetFlags implements modelcmd.Command.
+func (c *destroyControllerCommand) SetFlags(f *gnuflag.FlagSet) {
+	c.CommandBase.SetFlags(f)
+	c.out.AddFlags(f, "json", map[string]cmd.Formatter{
```

Comment:
hmm.. i'd have to look to confirm it, but i think `yaml` is usually the default for juju commands